cockatrice/src/gamesmodel.cpp

```diff
@@ -428,9 +445,9 @@ bool GamesProxyModel::filterAcceptsRow(int sourceRow) const
     if (!gameTypeFilter.isEmpty() && gameTypes.intersect(gameTypeFilter).isEmpty())
         return false;
 
-    if ((maxPlayersFilterMin != -1) && ((int)game.max_players() < maxPlayersFilterMin))
+    if ((maxPlayersFilterMin != DEFAULT_MAX_PLAYERS_MIN) && ((int)game.max_players() < maxPlayersFilterMin))
```

Comment:
This tests for -1 to allow you to turn it off, this change would mean that off is the default and that that specific amount would mean off. Probably it'd show 1 person games even though you set it to more than 1. It's pretty rare that you're looking for 1 person games so I think hiding them by default makes sense. This doesn't make sense for the max though, I think the default for those should be off instead of hiding the 99 player games.

Reply:
Gotcha! Updated.